src/encoder.rs

```diff
@@ -558,6 +558,9 @@ pub struct FrameInvariants {
     pub ref_frame_sign_bias: [bool; INTER_REFS_PER_FRAME],
     pub rec_buffer: ReferenceFramesSet,
     pub base_q_idx: u8,
+    pub y_dc_delta_q: i8,
+    pub uv_dc_delta_q: i8,
+    pub uv_ac_delta_q: i8,
```

Comment:
It looks like we're hard-coding the assumption that we're using the same deltas for Cb and Cr, but the spec allows them to be different (unlike VP9). That's probably fine as a first step in this patch, but we should probably relax that assumption before we go too far down the route of hard-coding it everywhere.